src/js/components/purchaseOrder/PurchaseOrderListTable.jsx

```diff
@@ -168,6 +168,47 @@ const PurchaseOrderListTable = ({
     Alert.error(translate('react.default.featureNotSupported', 'This feature is not currently supported'));
   };
 
+  const getStatusTooltip = (status) => {
```

Comment:
Instead of doing this, just add messages like:
`react.enum.OrderSummaryStatus.PENDING.description=This Purchase Order is in progress and has not been finalized.`
and do:
`translate(``react.enum.OrderSummaryStatus.${status}.description``)`